apostrophecms/form
# add form divider widget

## Summary

Add divider widget

## What are the specific steps to test this change?

1. Follow the readme instructions about the divider widget
2. Add the divider widget to a form
3. Check that the widget is displayed when not logged in

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
